easybuild/easyconfigs/i/IPM/IPM-2.0.6-CrayGNU-2016.11.eb

```diff
@@ -0,0 +1,67 @@
+easyblock = 'ConfigureMake'
+
+name = 'IPM'
+version = '2.0.6'
+# versionsuffix = '-debug'
+# cudaversion =  '8.0'
+# versionsuffix = '-cuda-%s' % cudaversion
+
+homepage = 'https://github.com/nerscadmin/IPM'
+description = """Integrated Performance Monitoring for High Performance
+Computing"""
+
+toolchain = {'name': 'CrayGNU', 'version': '2016.11'}
+toolchainopts = {'verbose': False}
+# https://github.com/hpcugent/easybuild-easyconfigs/pull/2928#issuecomment-216220341
```

Comment:
Please add this information as comment, then it will make sense to anyone reading the file.